### PR TITLE
Fix downgrade CI

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -58,21 +58,21 @@ ChainRulesCore = "1.18"
 CommonSolve = "0.2.4"
 ConstructionBase = "1.5"
 DataFrames = "1.6"
-Distributed = "1.9"
+Distributed = "1.10"
 DocStringExtensions = "0.9"
 EnumX = "1"
-FillArrays = "1.9"
+FillArrays = "1.10"
```
Comment (ChrisRackauckas) [inline Project.toml:64]:
```suggestion
FillArrays = "1.9"
```

File: Project.toml [hunk 2]
```diff
@@ -58,37 +58,37 @@ ChainRulesCore = "1.18"
...
 ModelingToolkit = "8.74"
 PartialFunctions = "1.1"
 PrecompileTools = "1.2"
 Preferences = "1.3"
-Printf = "1.9"
+Printf = "1.10"
 PyCall = "1.96"
 PythonCall = "0.9.15"
 RCall = "0.14.0"
 RecipesBase = "1.0"
```
Comment (ChrisRackauckas) [inline Project.toml:79]:
```suggestion
RecipesBase = "1.3.4"
```

File: Project.toml [hunk 3]
```diff
@@ -58,37 +58,37 @@ ChainRulesCore = "1.18"
...
 FunctionWrappersWrappers = "0.1.3"
 IteratorInterfaceExtensions = "^1"
-LinearAlgebra = "1.9"
-Logging = "1.9"
+LinearAlgebra = "1.10"
+Logging = "1.10"
 Makie = "0.20"
-Markdown = "1.9"
+Markdown = "1.10"
 ModelingToolkit = "8.74"
```
Comment (ChrisRackauckas) [inline Project.toml:71]:
```suggestion
ModelingToolkit = "8.75"
```

File: Project.toml [hunk 4]
```diff
@@ -58,37 +58,37 @@ ChainRulesCore = "1.18"
...
+Printf = "1.10"
 PyCall = "1.96"
 PythonCall = "0.9.15"
 RCall = "0.14.0"
-RecipesBase = "1.0"
-RecursiveArrayTools = "3.3.4"
+RecipesBase = "1.3.4"
+RecursiveArrayTools = "3.8.0"
 Reexport = "1"
 RuntimeGeneratedFunctions = "0.5"
```
Comment (ChrisRackauckas) [inline Project.toml:82]:
```suggestion
RuntimeGeneratedFunctions = "0.5.12"
```

File: Project.toml [hunk 5]
```diff
@@ -58,37 +58,37 @@ ChainRulesCore = "1.18"
...
 StaticArrays = "1.7"
 StaticArraysCore = "1.4"
-Statistics = "1.9"
+Statistics = "1.10"
 SymbolicIndexingInterface = "0.3.3"
 Tables = "1.11"
 TruncatedStacktraces = "1.4"
 Zygote = "0.6.67"
-julia = "1.9"
+julia = "1.10"
```
Comment (andreasnoack) [inline Project.toml:91]: Why is it necessary to bump the Julia requirement?

Comment (ChrisRackauckas) [reply Project.toml:91]: We just support the coming LTS. If anyone wants to decrease that, they would have to commit to supporting it for all of SciML.

**Pull request merged**